builds/misc/templates/mqtt-perf-run.yaml

```diff
@@ -4,262 +4,126 @@ jobs:
-      qos: 0
-  
-  ### QoS 1
-  
+      payload: 1024
+      qos: 1
+
   - template: mqtt-perf-test-case.yaml
     parameters: 
-      parentJob: run_completed_103
-      id: 150
+      parentJob: run_completed_6
+      id: 7
       publishers: 10
-      subscribers: 10
+      subscribers: 1
       topics: 1
-      payload: 32
-      qos: 1
-  
+      payload: 1024
+      qos: 0
+
   - template: mqtt-perf-test-case.yaml
     parameters: 
-      parentJob: run_completed_150
-      id: 151
+      parentJob: run_completed_7
+      id: 8
       publishers: 10
-      subscribers: 10
+      subscribers: 1
       topics: 1
-      payload: 131072 # 128kb
-      qos: 1
-  
-  - template: mqtt-perf-test-case.yaml
-    parameters: 
-      parentJob: run_completed_151
-      id: 152
-      publishers: 100
-      subscribers: 100
-      topics: 100
-      payload: 32 # 128kb
+      payload: 1024
       qos: 1
 
+## 1mb
+
   - template: mqtt-perf-test-case.yaml
     parameters: 
-      parentJob: run_completed_152
-      id: 153
+      parentJob: run_completed_8
+      id: 9
       publishers: 10
       subscribers: 10
       topics: 10
-      payload: 131072 # 128kb
-      qos: 1
-  
-  ################################################################################	
-  # Fan-out Test cases (few publishers/many subscribers)
-  ################################################################################	
-  
-  ### QoS 0
-  
-  - template: mqtt-perf-test-case.yaml
-    parameters: 
-      parentJob: run_completed_153
-      id: 200
-      publishers: 2
-      subscribers: 100
-      topics: 1
-      payload: 32
-      qos: 0
-  
-  - template: mqtt-perf-test-case.yaml
-    parameters: 
-      parentJob: run_completed_200
-      id: 201
-      publishers: 2
-      subscribers: 10
-      topics: 1
-      payload: 131072 # 128kb
+      payload: 1048576
       qos: 0
 
   - template: mqtt-perf-test-case.yaml
     parameters: 
-      parentJob: run_completed_201
-      id: 202
-      publishers: 2
-      subscribers: 100
-      topics: 100
-      payload: 32
-      qos: 0
-  
-  - template: mqtt-perf-test-case.yaml
-    parameters: 
-      parentJob: run_completed_202
-      id: 203
-      publishers: 2
+      parentJob: run_completed_9
+      id: 10
+      publishers: 10
       subscribers: 10
       topics: 10
-      payload: 131072 # 128kb
-      qos: 0
-  
-  ### QoS 1
-  
-  - template: mqtt-perf-test-case.yaml
-    parameters: 
-      parentJob: run_completed_203
-      id: 250
-      publishers: 2
-      subscribers: 100
-      topics: 1
-      payload: 32
+      payload: 1048576
       qos: 1
-  
+
   - template: mqtt-perf-test-case.yaml
     parameters: 
-      parentJob: run_completed_250
-      id: 251
-      publishers: 2
-      subscribers: 10
+      parentJob: run_completed_10
+      id: 11
+      publishers: 10
+      subscribers: 1
       topics: 1
-      payload: 131072 # 128kb
-      qos: 1
+      payload: 1048576
```

Comment:
1Mb messags with 10 publishers and 1 sub? Will this run fine? Won't it eventually choke?

Reply:
We'll see :)